lib/id3vx/frame/synchronised_lyrics_text.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Id3vx.Frame.SynchronisedLyricsText do
+  defstruct encoding: :utf16,
+            language: nil,
+            format: nil,
+            content_type: nil,
+            content_descriptor: nil
+
+  alias Id3vx.Frame.SynchronisedLyricsText
+
+  @type t :: %SynchronisedLyricsText{
+          encoding: Frame.text_encoding_v3() | Frame.text_encoding_v4(),
+          language: String.t(),
+          format: Frame.timestamp_format(),
+          content_type: Frame.content_type(),
+          content_descriptor: String.t()
```

Comment:
This is everything in the SYLT header but none of the actual lyrics or text.

From spec:

> The text that follows the frame header differs from that of the unsynchronised lyrics/text transcription in one major way. Each syllable (or whatever size of text is considered to be convenient by the encoder) is a null terminated string followed by a time stamp denoting where in the sound file it belongs. Each sync thus has the following structure:
> 
> Terminated text to be synced (typically a syllable) Sync identifier (terminator to above string) $00 (00) Time stamp $xx (xx ...)

I would call the field text_parts and use a list of tuples to do it.
```
[
  {0, "Strang"},
  {5, "ers"}
]
```

See the spec. I would finish the others before this frame. It is complex and not very important right now.